src/ray/gcs/gcs_server/gcs_resource_manager.h

```diff
@@ -205,7 +208,9 @@ class GcsResourceManager : public rpc::NodeResourceInfoHandler {
   absl::flat_hash_map<NodeID, SchedulingResources> cluster_scheduling_resources_;
   /// Placement group load information that is used for autoscaler.
   absl::optional<std::shared_ptr<rpc::PlacementGroupLoad>> placement_group_load_;
-
+  absl::flat_hash_map<NodeID, int64_t> latest_resources_normal_task_timestamp_;
```

Comment:
Please give some comments about what's this field for and how it's going to be used.